qt/python/mantidqt/utils/test/test_writetosignal.py

```diff
@@ -12,7 +12,7 @@
 import unittest
 
 import sys
-from mock import patch
```

Comment:
(I think) this will fail for Py3 because mock has moved, see https://github.com/mantidproject/mantid/blob/master/scripts/test/SANS/gui_logic/run_tab_presenter_test.py#L28 
Note also that, contrary to `run_tab_presenter_test`, we should generally do `if python2: do something` rather than if python3 to make the code more compatible with py4 when it eventually exists